### PR TITLE
Create 2019-04-19-coreweekly-week-15-2019.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 01st to Sunday 07th of April 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+The tests and debug phase of the 1.7.6 release has started: 134 issues have been created on the week after the code freeze. Some developers are already testing it on their infrastructure, opening issues, and submiting pull request, and that is AWESOME!!!
```
Comment (matks) [inline news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md]:
```suggestion
The tests and debug phase of the 1.7.6 release has started: 134 issues have been created on the week after the code freeze. Some developers are already testing it on their infrastructure, opening issues, and submitting pull requests, and that is AWESOME!!!
```

File: news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+Please continue like that, thank you very much for your feedback.
+
+Oh, and this week, I went to the first edition of the PSconnect event in Lyon, France, with a few people of the PrestaShop company: business managers, product managers, our CEO and our CTO. That was a very interesting moment to discuss and collect feedback about the roadmap. Merchants, partners, agencies and freelancers confirmed that the project's roadmap is on good tracks. There is a very good feedback about 1.7.5, with many projects on production and scheduled migrations.
```
Comment (matks) [inline news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md]:
```suggestion
Oh, and this week, I went to the first edition of the PSconnect event in Lyon, France, with a few people of the PrestaShop company: business managers, product managers, our CEO and our CTO. That was a very interesting moment to discuss and collect feedback about the roadmap. Merchants, partners, agencies and freelancers confirmed that the project's roadmap is on good tracks. There is a very good feedback about 1.7.5, with many projects in production and scheduled migrations.
```

File: news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+
+
+## Code changes in the "1.7.6.x" branch (for v1.7.6.0)
+
+### Core
+
+* [#13246](https://github.com/PrestaShop/PrestaShop/pull/13246): Add missing hook description, by [@eternoendless](https://github.com/eternoendless)
+* [#13256](https://github.com/PrestaShop/PrestaShop/pull/13256): Fixes to Customer subdomain & controller. Thank you [@sarjon](https://github.com/sarjon)
+* [#13266](https://github.com/PrestaShop/PrestaShop/pull/13266): Merge 1.7.6.x to develop - 10/04/2019, by [@matks](https://github.com/matks)
```
Comment (matks) [inline news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md]: This PR targeted the develop branch https://github.com/PrestaShop/PrestaShop/pull/13266

Also I think we can remove it as this is a "transfer PR" whose only goal is to retrieve commits from 1.7.6.x to develop

File: news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+* [#13232](https://github.com/PrestaShop/PrestaShop/pull/13232): [CO] Wrong calculation, free Gift. Thank you [@123monsite-regis](https://github.com/123monsite-regis)
+* [#13235](https://github.com/PrestaShop/PrestaShop/pull/13235): Update src folder license headers, by [@matks](https://github.com/matks)
+* [#13237](https://github.com/PrestaShop/PrestaShop/pull/13237): Update tests folders license headers, by [@matks](https://github.com/matks)
+* [#13238](https://github.com/PrestaShop/PrestaShop/pull/13238): Update themes folders license headers, by [@matks](https://github.com/matks)
+* [#13239](https://github.com/PrestaShop/PrestaShop/pull/13239): Update js folder license headers, by [@matks](https://github.com/matks)
+* [#13242](https://github.com/PrestaShop/PrestaShop/pull/13242): Add missing hook description for additionalCustomerAddressFields, by [@eternoendless](https://github.com/eternoendless)
+* [#13359](https://github.com/PrestaShop/PrestaShop/pull/13359): Merge 1.7.6.x to develop - 11/04/2019, by [@matks](https://github.com/matks)
```
Comment (matks) [inline news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md:81]: I think we can remove it as this is a "transfer PR" whose only goal is to retrieve commits from 1.7.6.x to develop

File: news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+* [#13375](https://github.com/PrestaShop/PrestaShop/pull/13375): Merge 1.7.6.x to develop - 12/04/2019, by [@matks](https://github.com/matks)
```
Comment (matks) [inline news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md:82]: I think we can remove it as this is a "transfer PR" whose only goal is to retrieve commits from 1.7.6.x to develop

File: news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 14 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-04-08 14:30:00
```
Comment (matks) [inline news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md]: We're the 19th of April are we not 🤔?

Comment (ttoine) [reply news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md]: header is updated, I was editing it while you wrote this comment.

File: news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md (hunk 5 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md:82]: I usually keep them. after all, this is work too

File: news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md (hunk 3 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md]: moved, thanks.

File: news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+## General messages
+
+Dear Developers,
+
+The tests and debug phase of the 1.7.6 release has started: 134 issues have been created on the week after the code freeze. Some developers are already testing it on their infrastructure, opening issues, and submitting pull requests, and that is AWESOME!!!
+
+Please continue like that, thank you very much for your feedback.
+
+Oh, and this week, I went to the first edition of the PSconnect event in Lyon, France, with a few people of the PrestaShop company: business managers, product managers, our CEO and our CTO. That was a very interesting moment to discuss and collect feedback about the roadmap. Merchants, partners, agencies and freelancers confirmed that the project's roadmap is on good tracks. There is a very good feedback about 1.7.5, with many projects in production and scheduled migrations.
```
Comment (MrBaiame) [inline news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md]: to discuss and collect feedback ==> feedbacks

Comment (ttoine) [reply news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md]: feedback is usually without s.

**Pull request merged**